corehq/apps/userreports/fixtures.py

```diff
@@ -51,16 +51,22 @@ def __call__(self, user, version, last_sync=None):
 
         root = ElementTree.Element('fixture', attrib={'id': self.id})
         reports_elem = ElementTree.Element('reports')
+        used_report_configs = []
         for report_config in report_configs:
             try:
+                report_config._index = len(filter(
+                    lambda used_report_config: report_config.report_id == used_report_config.report_id,
+                    used_report_configs
+                ))
                 reports_elem.append(self._report_config_to_fixture(report_config, user))
+                used_report_configs.append(report_config)
```

Comment:
You may want to move these lines out of the try/except block so that it's clear which line you're targeting. (thing's before the target line can go before `try:`, things after the target line can go in an `else:` block after the `except:` block.